cli/admin/src/test/java/io/pravega/cli/admin/controller/ControllerCommandsTest.java

```diff
@@ -130,44 +123,6 @@ public void testDescribeReaderGroupCommand() {
         Assert.assertNotNull(ControllerDescribeReaderGroupCommand.descriptor());
     }
 
-    @Test
-    @SneakyThrows
-    public void testDescribeStreamCommand() {
-        String scope = "testScope";
-        String testStream = "testStream";
-
-        String commandResult = executeCommand("controller describe-stream " + scope + " " + testStream, cliConfig());
-        Assert.assertTrue(commandResult.contains("stream_config"));
-        Assert.assertTrue(commandResult.contains("stream_state"));
-        Assert.assertTrue(commandResult.contains("segment_count"));
-        Assert.assertTrue(commandResult.contains("is_sealed"));
-        Assert.assertTrue(commandResult.contains("active_epoch"));
-        Assert.assertTrue(commandResult.contains("truncation_record"));
-        Assert.assertTrue(commandResult.contains("scaling_info"));
-
-        // Exercise actual instantiateSegmentHelper
-        CommandArgs commandArgs = new CommandArgs(Arrays.asList(scope, testStream), cliConfig());
-        ControllerDescribeStreamCommand command = new ControllerDescribeStreamCommand(commandArgs);
-        @Cleanup
-        CuratorFramework curatorFramework = CuratorFrameworkFactory.newClient(CLUSTER.zookeeperConnectString(),
-                new RetryOneTime(5000));
-        curatorFramework.start();
-        @Cleanup
-        ConnectionPool pool = new ConnectionPoolImpl(CLIENT_CONFIG, new SocketConnectionFactoryImpl(CLIENT_CONFIG));
-        @Cleanup
-        SegmentHelper sh = command.instantiateSegmentHelper(curatorFramework, pool);
-        Assert.assertNotNull(sh);
-
-        // Try the Zookeeper backend, which is expected to fail and be handled by the command.
-        Properties properties = new Properties();
-        properties.setProperty("cli.store.metadata.backend", CLIConfig.MetadataBackends.ZOOKEEPER.name());
-        cliConfig().getConfigBuilder().include(properties);
-        commandArgs = new CommandArgs(Arrays.asList(scope, testStream), cliConfig());
-        new ControllerDescribeStreamCommand(commandArgs).execute();
-        properties.setProperty("cli.store.metadata.backend", CLIConfig.MetadataBackends.SEGMENTSTORE.name());
-        cliConfig().getConfigBuilder().include(properties);
-    }
-
     static String executeCommand(String inputCommand, AdminCommandState state) throws Exception {
```

Comment:
You can remove these methods and classes since they were only used for the old test. The `executeCommand` method and the `TestingDescribeStreamCommand`.

Reply:
removed unused util methods